site/assets/scss/_subnav.scss

```diff
@@ -40,3 +40,7 @@
   line-height: 1;
   color: $gray-900;
 }
+
+.bd-search-docs-toggle:focus {
```

Comment:
Please move this in under the selector in line 39 and make sure CI passes.

Reply:
Fixed and commited again, all tests passed.